### PR TITLE
Fixes #15948 - Add in the engine initializer to extract strings

### PR DESCRIPTION
**Pull request by bkearney**

### CODE REVIEW

File: lib/foreman_ansible/engine.rb [hunk 1]
```diff
@@ -10,6 +13,13 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/services"]
     config.autoload_paths += Dir["#{config.root}/app/views"]
 
+    initializer 'foreman_docker.register_gettext', :after => :load_config_initializers do
```
Comment (agx) [inline lib/foreman_ansible/engine.rb]: Should this bee a foreman_ansible instead of foreman_docker ?

Comment (dLobatog) [inline lib/foreman_ansible/engine.rb]: Yeah, it doesn't really matter unless you have both foremandocker & foremanansible installed

**Comment by bkearney** [comment]
@agx sorry, bad copy/paste. I pushed a corrected initializer.

**Comment by dLobatog** [comment]
Merged as 537415c14d364c72518141b2727e7e24f50d9e1d, thanks @bkearney!

**Comment by bkearney** [comment]
@dLobatog thanks. Can you push to Transifex, or is that something @domcleal does?

**Comment by dLobatog** [comment]
@bkearney I don't think I have push permissions there, sorry :disappointed_relieved: !

**Pull request closed without merging**